flake.nix

```diff
@@ -0,0 +1,51 @@
+{
+  description = "A flake for s2n-tls";
+
+  inputs.nixpkgs.url = github:NixOS/nixpkgs/nixos-22.11;
+
+  outputs = { self, nix, nixpkgs, flake-utils }: 
+     flake-utils.lib.eachDefaultSystem (system:
+      let pkgs = nixpkgs.legacyPackages.${system};
+   in rec {
+      packages.s2n-tls = pkgs.stdenv.mkDerivation {
+        src = self;
+        name = "s2n-tls";
+        inherit system; 
+        # TODO: override the s2n-tls version
+
+        buildInputs = [ pkgs.cmake
+                        pkgs.openssl_3 ]; # s2n-config has find_dependency LibCrypto
```

Comment:
We should try and get AWS-LC to maintain a flake :)

Reply:
Totally part of the plan.  Unfortunately, the boringssl nixpkg doesn't look like it's in a state where we can copy/paste.